projects/templatetags/projects_extras.py

```diff
@@ -0,0 +1,80 @@
+from django import template
+from projects.models import *
+
+register = template.Library()
+
+
+@register.filter
+def get_bundles(project):
+    return BundleDetail.objects.filter(project__id=project.id).exclude(status='ARCHIVED')
+
+
+@register.filter
+def get_builds(bundle):
+    return BuildDetail.objects.filter(bundle__id=bundle.id).exclude(status='ARCHIVED')
+
+
+@register.filter
+def get_suite(bundle):
+    return SuiteDetail.objects.filter(bundle__id=bundle.id).exclude(status='ARCHIVED')[0]
```

Comment:
It really is just an extension of bundle and didn't really need its own model since it maps one-to-one, but it's easier for me to understand and keep track of views etc. If there is a better way to do this, let me know